backoffice/src/views/SingleLegacyBeaconRecordView.tsx

```diff
@@ -63,6 +72,30 @@ export const SingleLegacyBeaconRecordView: FunctionComponent<
 
   const hexId = beacon?.hexId || "";
   const numberOfUses = beacon?.uses?.length.toString() || "";
+  const reasonsForLegacyDeletion: string[] | undefined = Object.values(
```

Comment:
Is this duplicated across both legacy and modern, worth pulling out into a separate component?